tests/integration/targets/systemd_creds_decrypt/tasks/main.yaml

```
@@ -0,0 +1,58 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Test systemd_creds_decrypt
+  when:
+    - ansible_systemd.version is defined
+    - ansible_systemd.version | int >= 250
+  block:
+    - name: Encrypt secret
+      become: true
+      community.general.systemd_creds_encrypt:
```

Please use the short name for modules from this collection in integration tests.
```suggestion
      systemd_creds_encrypt:
```